src/Util/TestListenerTrait.php

```diff
@@ -109,8 +109,10 @@ function {$f['name']}{$f['signature']}
                 $mainSuite->addTest(new TestListener($suite));
             }
         }
-        foreach ($warnings as $w) {
-            $mainSuite->addTest($w);
+        if (isset($warnings)) {
```

Comment:
is this change still necessary, given that other changes in this file have been reverted ?

Reply:
reverted